src/oss/confluent.sh

```diff
@@ -911,7 +911,26 @@ list_command() {
     fi
 }
 
+validate_java_version() {
+    local target_service=${1}
+    if [[ $target_service = "zookeeper" || target_service = "kafka" ]]; then
+        return
+    fi
+    local java_version=$(java -version 2>&1 | sed -n ';s/.* version "\(.*\)\.\(.*\)\..*"/\1\2/p;')
```

Comment:
Are we sure that the output of `java -version` is going to match this regex for different JVMs?

Reply:
Nope. Wasn't sure how I could test this, so the code is designed to "fail safe" and just show a warning if the regex parse fails. It will only hard-fail if it does parse correctly and finds an illegal version. If you have any ideas on how to test/explore this on more platforms, I'd definitely prefer to incorporate

Reply:
Nah, that sounds sufficient. Couldn't find any specs on what the output of `java -version` should be across JVMs after a (brief) search so this seems like the right call.

Reply:
If you don't want to rely on the os-specific output of `java -version` command - you can also create and execute jar file which runs `System.getProperty("java.version")` but it seems like overkill here.

Reply:
Yes I think ideally we'd just put that in the `main` of each service to have a single point of truth, but this is meant to help our newbie 15-minute customers so this is sufficient.